R/derive_vars_computed.R

```diff
@@ -0,0 +1,194 @@
+#' Adds a Variable Computed from the Analysis Value of one or more Parameters
+#'
+#' Adds a variable computed from the analysis value of one or more parameters.
+#' It is expected that the value of the new variable is defined by an expression
+#' using the analysis values of other parameters. For example Body Mass Index at
+#' Baseline (`BMIBL`) in `ADSL` can be derived from of HEIGHT and WEIGHT
+#' parameters in `ADVS`.
+#'
+#' @param dataset
+#'
+#'   The variables specified by the `by_vars` parameter are expected.
+#'
+#' @param dataset_add Additional dataset
+#'
+#'   The variables specified by the `by_vars` parameter are expected.
+#'
+#'   The variable specified by `by_vars` and `PARAMCD` must be a unique key of
+#'   the additional dataset after restricting it by the filter condition
+#'   (`filter_add` parameter) and to the parameters specified by `parameters`.
```

Comment:
I think we should add in the need for `exprs` in the argument.  Can you look at other `by_vars` arguments and https://github.com/pharmaverse/admiral/pull/2250 to align

Reply:
Updated the *Permitted Values* for the `by_vars` argument